### ISSUE TITLE
Fix prop type errors in tests

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#376]
```
 PASS  packages/data-management/src/AddConnectionDrawerWidget/components/AddConnectionDrawerWidget.test.js (5.648s)                                                                                         
  ● Console                                                                                                                                                                                                 
                                                                                                                                                                                                            
    console.error node_modules/prop-types/checkPropTypes.js:20                                                                                                                                              
      Warning: Failed prop type: Invalid prop `connectionType` of type `object` supplied to `ConfigureConnection`, expected `string`.                                                                       
          in ConfigureConnection (at AddConnectionDrawerWidget.js:64)                                                                                                                                       
          in AddConnectionDrawerWidget (created by WithStyles(undefined))                                                                                                                                   
          in WithStyles(undefined) (at AddConnectionDrawerWidget.test.js:42)                                                                                                                                
```                                                                         

```
 PASS  packages/wiggle/src/DensityRenderer/components/WiggleRendering.test.js
  ● Console

    console.error node_modules/prop-types/checkPropTypes.js:20
      Warning: Failed prop type: The prop `features` is marked as required in `WiggleRendering`, but its value is `undefined`.                                                                             
          in WiggleRendering (at WiggleRendering.test.js:9)
```

**Comment by garrettjstevens** [comment]
Got the AddConnection one in d97f76744eb09e18bff7ad928e96b7b976609388

**Comment by cmdcolin** [comment]
Fixed by above and 14ae07c3 for now
